frontend/src/utils/format.ts

```diff
@@ -229,7 +229,7 @@ export const formatPercentage = (num: number) => {
 	return new Intl.NumberFormat(undefined, {
 		minimumFractionDigits: 2,
 		maximumFractionDigits: 2,
-	}).format(num * 100)
+	}).format(num)
```

Comment:
This function `formatPercentage` seems to be used in several places, did you check these were not broken by the change?